brain-marks.xcodeproj/xcshareddata/xcschemes/brain-marks.xcscheme

```diff
@@ -34,6 +34,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "de"
```

Comment:
Do you know what this change was? What setting you may have changed that made this change in the scheme?

Reply:
The only thing I can think of is when I was testing in the simulator, I went to edit scheme > options > app language and changed to German so I could make sure all necessary words were German when it ran. Could that be the reason it was added below the build configuration?

Reply:
Yeah I'm guessing that might be it. We'll leave it for now, it seems fine